File: Easy/Python/1816. Truncate Sentence.py
class Solution(object):
    def truncateSentence(self, s, k):
        words = s.split(' ')
        truncated = ''

        for i in range(k):
            truncated = truncated + words[i]
            if i + 1 < k:
                truncated = truncated + ' '

        return truncated

File: Easy/Java/1816. Truncate Sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        String[] words = s.split(" ");
        String truncated = "";
        
        for(int i = 0; i < k; i++){
            truncated += words[i];
            if(i + 1 < k)
                truncated += " ";
        }

        return truncated;
    }
}
